api/src/test/java/org/openmrs/module/patientqueueing/api/PatientQueueingServiceTest.java

```diff
@@ -310,4 +311,42 @@ public void generateVisitNumber_shouldNotThrowOutOfIndexExceptionWhenPreviousQue
 
 		patientQueueingService.generateVisitNumber(location, patient);
 	}
+
+    @Test
+    public void getPatientQueueListBySearchParams_shouldReturnPatientQueuesThatMatchesParameters() throws Exception {
+
+        PatientQueueingService patientQueueingService = Context.getService(PatientQueueingService.class);
+
+		String dateCreated = "2019-10-07 19:08:26";
```

Comment:
What is the difference between dateCreated and date? Do you really need a variable for this?